src/schema/v2/partner/BulkOperation/bulkUpdateArtworksMetadataMutation.ts

```diff
@@ -45,6 +45,7 @@ interface Input {
     medium?: string
     offer: boolean
     priceAdjustment?: number
+    displayPriceRange?: boolean
```

Comment:
Minor: could we keep this one on the same line it was placed before these changes? It was easier to spot it when alphabetically sorted.